django/db/models/fields/related.py

```diff
@@ -427,8 +427,8 @@ class ForeignObject(RelatedField):
     rel_class = ForeignObjectRel
 
     def __init__(self, to, on_delete, from_fields, to_fields, rel=None, related_name=None,
-            related_query_name=None, limit_choices_to=None, parent_link=False,
-            swappable=True, **kwargs):
+                 related_query_name=None, limit_choices_to=None, parent_link=False,
```

Comment:
I know about pep8. Django codebase is not following pep8 for all cases. Also, if you're adding or changing logic it should not contain formatting changes to look cleaner.